packages/zoe/src/contractSupport/types.js

```diff
@@ -49,3 +49,73 @@
  * @param {string} [rightHasExitedMsg]
  * @returns {string}
  */
+
+/**
+ * @typedef {Object} OfferToReturns
+ *
+ * The return value of offerTo is a promise for the userSeat for the
+ * offer to the other contract, and a promise (`deposited`) which
+ * resolves when the payout for the offer has been deposited to the `toSeat`
+ * @property {Promise<UserSeat>} userSeatPromise
+ * @property {Promise<AmountKeywordRecord>} deposited
+ */
+
+/**
+ * @typedef {Record<Keyword,Keyword>} KeywordKeywordRecord
+ *
+ * A mapping of keywords to keywords.
+ */
+
+/**
+ * @callback OfferTo
+ *
+ * Make an offer to another contract instance (labeled contractB below),
+ * withdrawing the payments for the offer from a seat in the current
+ * contract instance (contractA) and depositing the payouts in another
+ * seat in the current contract instance (contractA).
+ *
+ * @param {ContractFacet} zcf
+ *   Zoe Contract Facet for contractA
+ *
+ * @param {ERef<Invitation>} invitation
+ *   Invitation to contractB
+ *
+ * @param {KeywordKeywordRecord=} keywordMapping
+ *   Mapping of keywords used in contractA to keywords to be used in
+ *   contractB. Note that the pathway to deposit the payout back to
+ *   contractA reverses this mapping.
+ *
+ * @param {Proposal} proposal
+ *   The proposal for the offer to be made to contractB
+ *
+ * @param {ZCFSeat} fromSeat
+ *   The seat in contractA to take the offer payments from.
+ *
+ * @param {ZCFSeat} toSeat
+ *   The seat in contractA to deposit the payout of the offer to.
```

Comment:
Please say whether fromSeat and toSeat can be the same.  (I'm expecting the answer to be yes. If that's the case can toSeat be the default?)

Reply:
Yup, `fromSeat` and `toSeat` can be the same. I will make a comment saying so and will make the `fromSeat` a default.

Reply:
Done